dask_cuda/local_cuda_cluster.py

```diff
@@ -399,7 +422,7 @@ def new_worker_spec(self):
                 "plugins": {
                     CPUAffinity(
                         get_cpu_affinity(nvml_device_index(0, visible_devices))
-                    ),
+                    ) if not DASK_USE_ROCM else None,
```

Comment:
@younseojava IMO you could assume that an AMD GPU is available when someone uses this code.

So:

```py
) if False else None,
```